repository/src/main/java/org/apache/atlas/repository/graph/GraphHelper.java

```diff
@@ -1723,4 +1728,9 @@ public static String getDelimitedClassificationNames(Collection<String> classifi
 
         return ret;
     }
+
+    public static Boolean getHaslineage(AtlasElement element) {
```

Comment:
getEntityHasLineage does the same thing. Why two methods?

Reply:
make sense, removing the other/duplicate method in addendum commit